modules/dynamic-plugins/proc-topology-configure.adoc

```diff
@@ -0,0 +1,326 @@
+= Configuration
+
+== Viewing OpenShift routes
+To view OpenShift routes, you must grant read access to the routes resource in the Cluster Role:
```

Comment:
`ClusterRole`

Check the other files in this PR for instances of ClusterRole that should be `ClusterRole`.